Voter/AclVoter.php

```diff
@@ -51,7 +77,7 @@ public function supportsAttribute($attribute)
         return \is_string($attribute) && $this->permissionMap->contains($attribute);
     }
 
-    public function vote(TokenInterface $token, $subject, array $attributes)
+    private function internalVote(TokenInterface $token, $subject, array $attributes)
```

Comment:
let's call this one `doVote`, it's more common for us I think

Reply:
Done